tests/e2e/tests_runner.sh

```diff
@@ -112,6 +100,49 @@ main() {
 		"agent_image.bats" \
 		"agent_image_encrypted.bats" \
 		"${script_dir}/operator_tests.bats"
+
+}
+
+# Tests for CC with QEMU on SEV HW
+run_kata_qemu_sev_tests() {
+	local tests_passing="Test SEV unencrypted container launch success"
```

Comment:
If all tests are passing then you don't need to list them individually. I made that way on my initial implementation because two of the non-tee tests were failing.